test/e2e_node/hugepages_test.go

```diff
@@ -189,7 +189,7 @@ func getHugepagesTestPod(f *framework.Framework, limits v1.ResourceList, mounts
 }
 
 // Serial because the test updates kubelet configuration.
-var _ = SIGDescribe("HugePages [Serial] [Feature:HugePages][NodeSpecialFeature:HugePages]", func() {
+var _ = SIGDescribe("HugePages [Serial] [NodeSpecialFeature] [Feature:HugePages] [Feature:HugePages]", func() {
```

Comment:
What makes it special? We need to be careful with this one because if we don't exclude `NodeSpecialFeature` this will get run with the regular `[Feature:.*]` selected tests.

Reply:
This one already has a `Feature` tag, I'm just replacing the `NodeSpecialFeature` one.